File: Arcade/TheCore/Kotlin/appleBoxes.kt
fun appleBoxes(k: Int): Int {
    var redApples: Int = 0;
    var yellowApples: Int = 0;
    
    (1..k).forEach {
        if (it % 2 == 0) redApples += it * it;
        else yellowApples += it * it;
    }
    
    return redApples - yellowApples;
}

File: Arcade/TheCore/Javascript/appleBoxes.js
function appleBoxes(k) {
    let yellowApples = 0;
    let redApples = 0;
    
    for (let n = 1; n <= k; n++) {
        if (n % 2 == 0) redApples += n * n;
        else yellowApples += n * n;
    }
    
    return redApples - yellowApples;
}